bylaws/2-cf-membership.md

```diff
@@ -26,17 +26,20 @@ A **project** within the CF represents collaborative innovation in open-source t
 
 ## Project Representation
 
-**Diverse Voices, Unified Decision-Making:** Each project within the Commonhaus Foundation (CF) has its unique perspective. To ensure all projects have a say in the foundation's direction, each one appoints a representative to the [Extended Governance Committee (EGC)][egc]. These representatives are chosen according to the project's governance practices.
+**Diverse Voices, Unified Decision-Making:** Each project within the Commonhaus Foundation (CF) has its unique perspective. To ensure all projects have a say in the foundation's direction, each one appoints a representative to the [Extended Governance Committee (EGC)][egc]. These representatives are chosen according to the project's governance practices and do not have to be general members.
```

Comment:
> To ensure all projects have

"To ensure that every project has"

> each one

"each project"

> These representatives are chosen

The representative is chosen


Yeah yeah, I know, I'm _just_ sufficiently self-aware to know when I'm being completely anal.

Reply:
I will throw it back at you: use the +- button and actually make the suggestion I can merge. ;)

Reply:
OMG where is this button I've been trying to find it for months I have seen people do this and it just looks like magic.

Reply:
I found it. It's essentially an easter egg. I don't know why they didn't just protect it behind the konami code.